src/frequenz/sdk/timeseries/battery_pool/battery_pool.py

```diff
@@ -0,0 +1,142 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""User interface for requesting aggregated battery-inverter data."""
+
+import asyncio
+from datetime import timedelta
+from typing import Any, Dict, Optional, Set
+
+from frequenz.channels import Receiver
+
+from ... import microgrid
+from ..._internal._constants import RECEIVER_MAX_SIZE
+from ..._internal.asyncio import cancel_and_await
+from ...actor.power_distributing._battery_pool_status import BatteryStatus
+from ...microgrid.component import ComponentCategory
+from ._methods import BaseMethod, ResamplerArgs, SendOnUpdate
+from ._metric_calculator import PowerBoundsCalculator, SoC_Capacity_Calculator
+from ._result_types import PowerBounds, SoC_Capacity
+
+
+class BatteryPool:
+    """Calculate high level metrics for a pool of the batteries.
+
+    BatterPool provided methods for fetching different battery and inverter values
+    from microgrid and aggregate then in a high level metrics.
+    Depending from the metrics the results are send in two ways:
+        * when the metric change
+        * with the resampling frequency (specified in the resampler actor outside)
```

Comment:
I think this won't render nicely if it has the extra indent, also a new line makes it more clear.

```suggestion

    * when the metric change
    * with the resampling frequency (specified in the resampler actor outside)
```

Reply:
Removed. We have only metrics sending on update in this PR.